example/src/Main.purs

```diff
@@ -72,7 +72,7 @@ ui = parentComponent render eval
                         )
                         Nothing
                 , initialState :
-                    { key : Nothing, editor : Nothing }
+                    initialAceState
```

Comment:
I'd like to try that, but I can't figure out how to transform the `SlotConstructor` that `aceConstructor` returns to an `HTML` element.